packages/graphql/tests/tck/directives/auth/arguments/bind/interface-relationships/implementation-bind.test.ts

```diff
@@ -264,18 +264,13 @@ describe("Cypher Auth Allow", () => {
             CALL {
             	 WITH this
             WITH this
-            OPTIONAL MATCH (this)-[this_has_content0_relationship:HAS_CONTENT]->(this_content0:Comment)
+            MATCH (this)-[this_has_content0_relationship:HAS_CONTENT]->(this_content0:Comment)
             WHERE this_content0.id = $updateUsers_args_update_content0_where_Commentparam0
-            CALL apoc.do.when(this_content0 IS NOT NULL, \\"
             WITH this, this_content0
```

Comment:
This block must be a subquery:

https://github.com/neo4j/graphql/pull/2752/files#diff-115f271fbaa3e70c8083dd3fde505e1dee6d583f737e5bf58225bfd1e367a3e5R269-R273